### ISSUE TITLE
Explicit re-export is not working

### ISSUE THREAD

**Issue by InakiAbete-Reflektion** [asapach/babel-plugin-rewire-exports#5]
I have 3 files:

### module1.js
```javascript
export const say = function (str) {
  console.log(str);
};
```

### module2.js
```javascript
import { say } from './module1';

export default 1;
export { say };
```

### script.js
```javascript
import { say } from './module2';

console.log('say', say);
say('hello world');
```

If I run `script.js` with `es2015`, `react` and `stage-3` babel presets and `rewire-exports` only in plugins, I get this error:
```
(0, _module.say)('hello world');
                ^

TypeError: (0 , _module.say) is not a function
```
(`_module.say` is `undefined`)

And this is the output of all those files running this command:
`babel --presets "es2015,react,stage-3" --plugins "rewire-exports" ./[FILE]`

### module1.js
```javascript
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
var say = exports.say = function say(str) {
  console.log(str);
};

```

### module2.js
```javascript
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.say = exports.default = undefined;
exports.rewire = rewire;
exports.rewire$say = rewire$say;
exports.restore = restore;

var _module = require('./module1');

var _default = 1;
exports.default = _default;
exports.say = _say;
var _say = _module.say;
var _default2 = _default,
    _say2 = _say;

function rewire($stub) {
  exports.default = _default = $stub;
}

function rewire$say($stub) {
  exports.say = _say = $stub;
}

function restore() {
  exports.default = _default = _default2;
  exports.say = _say = _say2;
}

```

### script.js
```javascript
'use strict';

var _module = require('./module2');

console.log('say', _module.say);
(0, _module.say)('hello world');

```

Could be these lines in `module2.js` output?
```javascript
...
var _default = 1;
exports.default = _default;
exports.say = _say;
var _say = _module.say;
var _default2 = _default,
    _say2 = _say;
...
```

`_say` seems not defined at the moment it's assigned to `exports.say`, but if I move that line up it works:
```javascript
...
var _default = 1;
exports.default = _default;
var _say = _module.say;
exports.say = _say;
var _default2 = _default,
    _say2 = _say;
...
```

**Comment by InakiAbete-Reflektion** [comment]
I'm definitely not sure if this it's the way to fix it, but seems working if I change the [line 176](https://github.com/asapach/babel-plugin-rewire-exports/blob/master/src/index.js#L176) from
```javascript
path.insertAfter(t.variableDeclaration('var', [t.variableDeclarator(id, local)]));
```
to
```javascript
path.insertBefore(t.variableDeclaration('var', [t.variableDeclarator(id, local)]));
```

**Comment by asapach** [comment]
Yes, it looks like you're right. I think I didn't notice it earlier because I'm mostly using webpack, which supports live bindings properly, so it didn't manifest there. I'll publish a new release shortly.

**Comment by asapach** [comment]
Released as [v0.3.0](https://github.com/asapach/babel-plugin-rewire-exports/releases/tag/v0.3.0)

**Comment by InakiAbete-Reflektion** [comment]
Thank you so much for this! You're a life saver.
